deploy/olm-catalog/operand-deployment-lifecycle-manager/1.2.1/operand-deployment-lifecycle-manager.v1.2.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,569 @@
+    mediatype: image/png
+  install:
+    spec:
+      clusterPermissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          - services
+          - services/finalizers
+          - endpoints
+          - persistentvolumeclaims
+          - events
+          - configmaps
+          - secrets
+          - namespaces
+          verbs:
+          - '*'
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - daemonsets
+          - replicasets
+          - statefulsets
+          verbs:
+          - '*'
+        - apiGroups:
+          - apps
+          resourceNames:
+          - operand-deployment-lifecycle-manager
+          resources:
+          - deployments/finalizers
+          verbs:
+          - update
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          verbs:
+          - get
+          - list
+        - apiGroups:
+          - apps
+          resources:
+          - replicasets
+          - deployments
+          verbs:
+          - get
+        - apiGroups:
+          - operator.ibm.com
+          - operators.coreos.com
+          resources:
+          - '*'
+          - operandregistries
+          - operandconfigs
+          - operandrequests
+          - operandbindinfos
+          verbs:
+          - '*'
```

Comment:
you can remove line 406~461, all these permissions are covered by following one.